### DISCUSSION TITLE
A new student in remix

### DISCUSSION THREAD

**Discussion by Oyennz** [epicweb-dev/epic-stack#889] [General]
Remix is new to me, but I want to work on a major project.
I saw that your project doesn't include _index.tsx, which I know is necessary for the default page in remix. How do you set this up?
or you might recommend a YouTube video you create for the setup process.

**Comment by lewisblackburn** [comment]
The marketing folder contains the landing page :)

**Reply by Oyennz** [reply]
may i know, how to know from root.tsx to this _marketing+>index.js cause i dont see any code that redirect to this?
![Capture](https://github.com/user-attachments/assets/46891258-5a64-4341-a798-9e9238db6f4b)

**Reply by lewisblackburn** [reply]
The symbols in the folder names .e.g. the underscore affect the behaviour. The underscore omits the "marketing" from the url. I'm not sure I can explain correctly, but the remix documentation site will have some information on this.

**Reply by kiliman** [reply]
Epic Stack is using the `remix-flat-routes` package. This was the initial implementation of Remix v2 flat routes and includes some extensions like hybrid routing (the `+` suffix).

NOTE: Index routes in Remix v2 _must_ be named `_index.tsx`. This is optional in `remix-flat-routes` as the v2 restriction was added after my package was released.

**Reply by Oyennz** [reply]
I see, thank you so much
